snapcraft/internal/lifecycle.py

```diff
@@ -336,8 +337,16 @@ def _tar_filter(tarinfo):
 
 def containerbuild(step, project_options, output=None, args=[]):
     config = snapcraft.internal.load_config(project_options)
+    remote = os.environ.get('SNAPCRAFT_CONTAINER_BUILDS')
+    # Default remote if it's a truthy value - otherwise it's a remote name
+    try:
+        util.strtobool(remote)
+        remote = None
+    except ValueError:
+        pass
```

Comment:
Exactly. Quoting myself from the exact same question asked by @elopio in this same PR: 

> This is so you can do SNAPCRAFT_CONTAINER_BUILDS=myremote to specify a remote or SNAPCRAFT_CONTAINER_BUILDS=1 to use the default (local) remote.

It's either a name or a truthy value. But this isn't where container builds are enabled or disabled.